conf/modules/call_svs.config

```diff
@@ -18,20 +18,39 @@ process {
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     */
 
+    withName: '.*:CALL_SVS:.*' {
+        publishDir = [
+            enabled: false,
+        ]
+    }
+
     withName: '.*:CALL_SVS:SNIFFLES' {
-        ext.prefix = { "${meta.id}_sniffles" }
+        ext.prefix = { "${meta.id}_sniffles_for_reheader" }
         ext.args = '--no-qc'
+    }
+
+    withName: '.*:CALL_SVS:BCFTOOLS_MERGE' {
+        ext.prefix = { "${meta.id}_svs" }
+        ext.args = [
+            '--force-single',
+            '--output-type z',
+            '--write-index=tbi'
+        ].join(' ')
         publishDir = [
-            path: { "${params.outdir}/sv_calling/sniffles/single_sample/${meta.id}" },
+            path: { "${params.outdir}/svs/multi_sample/${meta.id}" },
```

Comment:
do you need to have the `multi_sample` subdirectory, won't the meta.id after the merge be the case or project?

Reply:
For now I think yes, because I want to be able to `ls` or find in the directory and not mix VCFs from single samples with multisample vcfs.

Reply:
So it becomes

```
├── multi_sample
│   └── test_project
│       ├── test_project_svs.vcf.gz
│       └── test_project_svs.vcf.gz.tbi
└── single_sample
    ├── HG002_Revio
    │   ├── HG002_Revio_sniffles.vcf.gz
    │   └── HG002_Revio_sniffles.vcf.gz.tbi
    ├── HG002_Revio_A_copy
    │   ├── HG002_Revio_A_copy_sniffles.vcf.gz
    │   └── HG002_Revio_A_copy_sniffles.vcf.gz.tbi
    ├── HG002_Revio_B_copy
    │   ├── HG002_Revio_B_copy_sniffles.vcf.gz
    │   └── HG002_Revio_B_copy_sniffles.vcf.gz.tbi
    └── HG002_Revio_C_copy
        ├── HG002_Revio_C_copy_sniffles.vcf.gz
        └── HG002_Revio_C_copy_sniffles.vcf.gz.tbi
```

and not
```
└── svs
    └── test_project
    │   ├── test_project_svs.vcf.gz
    │   └── test_project_svs.vcf.gz.tbi
    ├── HG002_Revio
    │   ├── HG002_Revio_sniffles.vcf.gz
    │   └── HG002_Revio_sniffles.vcf.gz.tbi
    ├── HG002_Revio_A_copy
    │   ├── HG002_Revio_A_copy_sniffles.vcf.gz
    │   └── HG002_Revio_A_copy_sniffles.vcf.gz.tbi
    ├── HG002_Revio_B_copy
    │   ├── HG002_Revio_B_copy_sniffles.vcf.gz
    │   └── HG002_Revio_B_copy_sniffles.vcf.gz.tbi
    └── HG002_Revio_C_copy
        ├── HG002_Revio_C_copy_sniffles.vcf.gz
        └── HG002_Revio_C_copy_sniffles.vcf.gz.tbi
```

Could rename `multi_sample` and `single_sample` to something better though.

Reply:
Fair enough. The names are fairly descriptive already :)